libnymea/loggingcategories.h

```diff
@@ -51,6 +51,7 @@ Q_DECLARE_LOGGING_CATEGORY(dcAvahi)
 Q_DECLARE_LOGGING_CATEGORY(dcUpnp)
 Q_DECLARE_LOGGING_CATEGORY(dcBluetooth)
 Q_DECLARE_LOGGING_CATEGORY(dcCloud)
+Q_DECLARE_LOGGING_CATEGORY(dcCloudTraffic)
```

Comment:
Should be added to the main.cpp of the server, so the list of available debug categories i complete and explicitly disable it for default.

Reply:
thanks! added.